src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Cat;
+import core.basesyntax.model.Dog;
+import java.time.LocalDate;
+
+public class HelloWorld {
+    private String str;
```

Comment:
```suggestion
    private String greeting;
```

it is incorrect to use shortened names like `str`, `u` instead of `user`, and so on